build.gradle

```diff
@@ -59,6 +60,10 @@ dependencies {
 	implementation 'org.zalando:logbook-spring-boot-starter:1.11.0'
 	implementation 'javax.validation:validation-api:2.0.1.Final'
 	implementation 'org.mapstruct.extensions.spring:mapstruct-spring-annotations:0.0.2'
+	implementation 'com.giffing.bucket4j.spring.boot.starter:bucket4j-spring-boot-starter:0.2.0'
```

Comment:
debiilselt palju dependenceid juurde, aga ma võtan selle tradeoffi plug and play lahenduse vastu